backup/Dockerfile

```diff
@@ -21,23 +18,24 @@ RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | \
                 barman \
                 postgresql-client-10
 
+RUN pip3 install \
+        docopt \
+        yacron
+
+RUN git clone https://github.com/vimc/montagu-metrics-py /tmp/metrics_utils && \
+        git -C /tmp/metrics_utils reset --hard 4b2ef9b && \
```

Comment:
Why is this here?

Reply:
To get the metrics code into the docker image. The other way is to add a submodule, do a `COPY` in the dockerfile and manage versions that way. Given submodules are a source of grief and we never need that code in the non-docker part of this, I thought we could try this approach